client/app/billing/autoRenew/autorenew.controller.js

```diff
@@ -197,4 +198,12 @@ export default class AutorenewCtrl {
 
     return `${RENEW_URL[this.currentUser.ovhSubsidiary]}${urlParameterDomains}`;
   }
+
+  getAutomaticExpirationDate(service) {
+    return _.capitalize(
+      new Intl.DateTimeFormat(
```

Comment:
or the angular way : 
`this.$filter('date')(new Date(...), 'medium')`
since we import angular-i18n it'll be internationalized

Reply:
Yes but this imply adding another link to `angular` and `filter`, I wanted to avoid this

Reply:
I don't know which one is best. I wanted to avoid having a dependency to `$filter` but there's a dependency to `angular-translate`

Reply:
I like the native way better, even though we should abstract it so we don't have to ` this.$translate.use().replace('_', '-')` and `{ year: 'numeric', month: 'long' }` each time :)

Reply:
The `{ year: 'numeric', month: 'long' } ` is specific to this need (we want to display only Month and yet)
Would it make sense to add a converter in a `manager-core` translate service ?

Reply:
I'm not convinced about have such a service in manager-core.

IMO we can keep it as is.